enginetest/priv_auth_queries.go

```diff
@@ -1662,4 +1662,242 @@ var QuickPrivTests = []QuickPrivilegeTest{
 			"UPDATE mydb.test SET v1 = 0;",
 		},
 	},
+	{
+		Queries: []string{
+			"FLUSH PRIVILEGES;",
+		},
+		ExpectingErr: true,
+	},
+	{
+		Queries: []string{
+			"GRANT RELOAD ON *.* TO tester@localhost",
+			"FLUSH PRIVILEGES;",
+		},
+	},
+}
+
+// PersistPrivilegesTest test the persist function in the grant tables, and we'll run an INSERT, UPDATE,
+// or DELETE on one of the privilege tables (mysql.user, mysql.db, etc.), modifying the privileges in some way.
+type PersistPrivilegesTest struct {
+	Name        string
+	SetUpScript []string
+	InOneServer []PersistPrivilegesTestInOneServer
+}
+
+// PersistPrivilegesTestInOneServer test persist assertions with restarting the server with the same privileges file
+type PersistPrivilegesTestInOneServer struct {
+	Assertions []PersistPrivilegesTestAssertion
+}
+
+// PersistPrivilegesTestAssertion is within a PersistPrivilegesTestInOneServer to assert functionality.
+type PersistPrivilegesTestAssertion struct {
+	User           string
+	Host           string
+	Query          string
+	Expected       []sql.Row
+	ExpectedErr    *errors.Kind
+	ExpectedErrStr string
+}
+
+// PersistPrivilegesTests test persistence of privilege systems. These tests always have the root account available, and the root
+// account is used with any queries in the SetUpScript.
+var PersistPrivilegesTests = []PersistPrivilegesTest{
+	{
+		Name: "Privilege testing on mysql.db table of FLUSH PRIVILEGES",
+		SetUpScript: []string{
+			"CREATE USER tester@localhost;",
+		},
+		InOneServer: []PersistPrivilegesTestInOneServer{
+			{
+				Assertions: []PersistPrivilegesTestAssertion{
+					{
+						User:     "root",
```

Comment:
Every assertion uses `User: "root"` and `Host: "localhost"`, so they should be omitted, as they dramatically increase the size of the assertions and make it harder to parse the tests. As with the other comments, make sure every line is included for a reason.

This is just informational, and as in the other comment, we don't need these tests.